### PR TITLE
Japanese localization

### PR DESCRIPTION
**Pull request by bnsports**

### CODE REVIEW

File: fearless/Common/Helpers/AssetBalanceFormatterFactory.swift [hunk 1]
```diff
@@ -2,6 +2,11 @@ import Foundation
 import SoraFoundation
 import SSFModels
 
+enum FormatterLocale {
```
Comment (DRadmir) [inline fearless/Common/Helpers/AssetBalanceFormatterFactory.swift:5]: I suggest create init from Locale.identifire

File: fearless/Common/Helpers/AmountFormatterFactory.swift [hunk 2]
```diff
@@ -79,53 +84,104 @@ struct AmountFormatterFactory: NumberFormatterFactoryProtocol {
...
-                threshold: 0,
-                divisor: 1.0,
-                suffix: "",
-                formatter: DynamicPrecisionFormatter(
-                    preferredPrecision: UInt8(preferredPrecision),
-                    roundingMode: roundingMode
+        let abbreviations: [BigNumberAbbreviation]
+        switch locale {
+        case .japanese:
+            abbreviations = [
```
Comment (DRadmir) [inline fearless/Common/Helpers/AmountFormatterFactory.swift:93]: It is advisable create struct with abbreviations arrays in other file
struct Abbreviations {
static let japanese = []
static let usual = []
}

File: fearless/Modules/Staking/SelectValidatorsFlow/RecommendedValidatorList/Flow/Pool/RecommendedValidatorListPoolViewModelFactory.swift [hunk 3]
```diff
@@ -24,13 +24,9 @@ class RecommendedValidatorListPoolViewModelFactory {
...
-    private func createItemsCountString(for currentCount: Int, outOf maxCount: Int) -> LocalizableResource<String> {
-        LocalizableResource { locale in
-            R.string.localizable.stakingSelectedValidatorsCount_v191(
-                currentCount,
-                maxCount,
-                preferredLanguages: locale.rLanguages
-            )
+    private func createItemsCountString(for _: Int, outOf _: Int) -> LocalizableResource<String> {
+        LocalizableResource { _ in
+            ""
```
Comment (DRadmir) [inline fearless/Modules/Staking/SelectValidatorsFlow/RecommendedValidatorList/Flow/Pool/RecommendedValidatorListPoolViewModelFactory.swift:29]: please check this

File: fearless/Modules/Staking/SelectValidatorsFlow/RecommendedValidatorList/Flow/Relaychain/RecommendedValidatorListRelaychainViewModelFactory.swift [hunk 4]
```diff
@@ -24,13 +24,9 @@ class RecommendedValidatorListRelaychainViewModelFactory {
...
-    private func createItemsCountString(for currentCount: Int, outOf maxCount: Int) -> LocalizableResource<String> {
-        LocalizableResource { locale in
-            R.string.localizable.stakingSelectedValidatorsCount_v191(
-                currentCount,
-                maxCount,
-                preferredLanguages: locale.rLanguages
-            )
+    private func createItemsCountString(for _: Int, outOf _: Int) -> LocalizableResource<String> {
+        LocalizableResource { _ in
+            ""
```
Comment (DRadmir) [inline fearless/Modules/Staking/SelectValidatorsFlow/RecommendedValidatorList/Flow/Relaychain/RecommendedValidatorListRelaychainViewModelFactory.swift:29]: please check this

File: fearless/Modules/Staking/StakingPayoutConfirmation/ViewModel/StakingPayoutConfirmViewModelFactory.swift [hunk 5]
```diff
@@ -42,7 +42,7 @@ import IrohaCrypto
 //
 //        return LocalizableResource { locale in
 //            let title = R.string.localizable
-//                .accountInfoTitle(preferredLanguages: locale.rLanguages)
+//                .commonAccount(preferredLanguages: locale.rLanguages)
```
Comment (DRadmir) [inline fearless/Modules/Staking/StakingPayoutConfirmation/ViewModel/StakingPayoutConfirmViewModelFactory.swift:45]: comments

**Review by DRadmir — CHANGES_REQUESTED** [review]

**Review by Viscount88 — APPROVED** [review]

**Pull request closed without merging**